recipe/meta.yaml

```
@@ -1,17 +1,17 @@
-{% set version = "1.1.8" %}
+{% set version = "1.2.2" %}
 
 package:
   name: dateparser
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/d/dateparser/dateparser-{{ version }}.tar.gz
-  sha256: 86b8b7517efcc558f085a142cdb7620f0921543fcabdb538c8a4c4001d8178e3
+  sha256: 986316f17cb8cdc23ea8ce563027c5ef12fc725b6fb1d137c14ca08777c5ecf7
 
 build:
   number: 0
-  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
-  skip: True  # [py<37]
+  skip: True  # [py<39]
```

https://github.com/scrapinghub/dateparser/blob/v1.2.2/setup.py#L48
```suggestion
  skip: True  # [py<38]
```